src/types/tags.ts

```diff
@@ -1,4 +1,4 @@
-export type ActivityDirectiveTagsInsertInput = {
+export type ActivityDirectiveTagsUpdateInput = {
```

Comment:
What is the reason for the renaming? The goal of using "insert" is to match more closely to the input query type in the GQL schema: https://github.com/NASA-AMMOS/aerie-ui/blob/9cabbcf50e3446dc12d5750f24e275c68277f907/src/utilities/gql.ts#L53 ("activity_directive_tags_insert_input")

Reply:
Ah I think I renamed this back when I misunderstood how these tags would be handled, fixing.